packages/apollo-server-core/src/playground.ts

```diff
@@ -10,7 +10,7 @@ export {
 // This specifies the version of `graphql-playground-react` that will be served
```

Comment:
Maybe the comments here should include `@apollographql/`?

Reply:
https://github.com/apollographql/apollo-server/pull/4932#discussion_r581718804